opentelemetry-otlp/tests/integration_test/Cargo.toml

```diff
@@ -9,10 +9,7 @@ publish = false
 once_cell = { workspace = true }
 opentelemetry = { path = "../../../opentelemetry", features = ["metrics", "logs"] }
 opentelemetry_sdk = { path = "../../../opentelemetry-sdk", features = ["rt-tokio", "logs", "testing"] }
-opentelemetry-otlp = { path = "../../../opentelemetry-otlp", features = ["tonic", "metrics", "logs"] }
-opentelemetry-semantic-conventions = { path = "../../../opentelemetry-semantic-conventions" }
-opentelemetry-proto = { path = "../../../opentelemetry-proto", features = ["gen-tonic-messages", "trace", "with-serde"] }
-opentelemetry-appender-log = { path = "../../../opentelemetry-appender-log", default-features = false}
+opentelemetry-proto = { path = "../../../opentelemetry-proto", features = ["gen-tonic-messages", "trace", "logs", "with-serde"] }
```

Comment:
Added `"logs"` to the list of features as it is needed for [log_asserter](https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry-otlp/tests/integration_test/src/logs_asserter.rs#L1).